### PR TITLE
Feature/add soilnox

### LINKED ISSUE #11: Time zone config
[It seems](https://web.archive.org/web/20200807020453/http://wiki.seas.harvard.edu/geos-chem/index.php/HEMCO_versions#Ocean_grid_boxes_now_use_the_timezone_of_the_nearest_land_mass_for_computing_emissions) that the timezone file specified in `NEXUS_Config.rc` is not the recommended one. The recommended `timezones_voronoi_1x1.nc` allegedly does better for coastal grid cells that have ocean overlap. Has this been tried in NEXUS?

### PR DESCRIPTION
**Pull request by bbakernoaa**
Feature to add soilnox 

Closes #11

### CODE REVIEW

**Comment by zmoon** [comment]
Closes #11

**Review by zmoon — COMMENTED** [review]
From what I can see the changes in concat post split and gfs bio scripts aren't needed. Could just restore them to keep the history cleaner.

File: utils/python/concatenate_nexus_post_split.py [hunk 1]
```diff
@@ -4,6 +4,15 @@
...
 """
 
+def dt_fmt(dt):
+     """Convert (cftime) datetime to string."""
+     Y = dt.year
+     m = dt.month
+     d = dt.day
+     H = dt.hour
+     M = dt.minute
+     return f"{Y:04d}-{m:02d}-{d:02d} {H:02d}:{M:02d}"
```
Comment (zmoon) [inline utils/python/concatenate_nexus_post_split.py]: (This is a dupe)

File: utils/combine_ant_bio.py [hunk 2]
```diff
@@ -151,7 +157,10 @@ def main(ifp, ofp, *, compress=True):
                 ds[f"{spc}_ant"][:]
                 + (ds["MTPA_bio"][:] + ds["MTPO_bio"][:] + ds["LIMO_bio"][:]) * 0.4666
             )
-
+        # elif spc == "NO":
+        #     ds_new[spc][:] = ds[f"{spc}_ant"][:] + ds["SOIL_NOx"][:]
```
Comment (zmoon) [inline utils/combine_ant_bio.py]: Remove?
```suggestion
```

**Review by zmoon — APPROVED** [review]
Config sounds fine to me.

But just curious, what is the implication/effect that the new one is emission category 2 but all the others are -1?

**Comment by bbakernoaa** [comment]
> Config sounds fine to me.
> 
> But just curious, what is the implication/effect that the new one is emission category 2 but all the others are -1?

It put it into a new category to keep it separate from NO emissions from the anthropogenic. Was having issues if we didn't

**Pull request merged**